ui/src/dashboard/menus/MenuForm.tsx

```diff
@@ -44,23 +46,51 @@ export const MenuForm = ({ onSubmit, menu = null }) => {
         // }}
         onSubmit={onSubmit}
       >
-        {({ isSubmitting, values }) => (
-          <>
-            <div className="card card--full-width card--lightestGray">
-              <Form>
-                <Field isFormik name="title" label="Title" />
-                <Field isFormik as="textarea" label="Description" name="description" />
-                <Field isFormik as="textarea" label="Disclaimer" name="disclaimer" />
-                <Button type="submit" disabled={isSubmitting}>
-                  {menu ? 'Update' : 'Create'} Menu
-                </Button>
-              </Form>
-            </div>
-            <div className="margin-top-small">
-              <MenuCategoryForm values={values} initialCategory={initialCategory} initialItem={initialItem} />
-            </div>
-          </>
-        )}
+        {({ isSubmitting, values, setFieldValue }) => {
+          console.log(values, 'values')
+          return (
+            <>
+              <div className="card card--full-width card--lightestGray">
+                <Form>
+                  <div className="grid-container grid">
+                    <div className="left">
+                      <p className="menu-header-text">Header Image</p>
+                      <Dropzone name="bgImage" setFieldValue={setFieldValue} />
+                    </div>
+                    <div className="right">
```

Comment:
same comment as left. Also, is there a way to use the Grid or Flex layout utility instead of custom right/left?